src/VectorizeLoops.cpp

```diff
@@ -563,6 +605,12 @@ class VectorSubs : public IRMutator {
         return mutate_binary_operator(op);
     }
 
+    Expr visit(const Ramp *op) override {
```

Comment:
This looks the same as the base-class implementation

Reply:
Removed.